README.md

```diff
@@ -1 +1,126 @@
-# Currency-Exchange
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Currency Exchange ](#-currency-exchange-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [Live Demo ](#live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Currency Exchange <a name="about-project"></a>
+
+- Currency Exchange is a web application that facilitates currency exchange between various countries. Users can choose any currency and convert it into another currency supported by the system. Additionally, the app provides details regarding currency codes and the corresponding country names linked to each currency.
+
+## 🛠 Built With <a name="HTML, CSS, JS and REACT"></a>
+
+### Tech Stack <a name="Front end (REACT)"></a>
+
+- This is a mobile and desktop browser responsive website.
+
+  Client
+  
+- HTML, CSS, JS, REACT
+- Technologies Used- HTML, CSS, JS, REACT, GIT, Linters (GitHub Actions, Lighthouse, Webhint, Stylelint, Eslint)
+
+### Key Features <a name="key-features"></a>
+
+- Provide cards with flag
+- Provide latest prices conversion
+- Provide countryName & currency
+- Provide other information 
+
+<!-- LIVE DEMO -->
+
+## Live Demo <a name="live-demo"></a>
+
+
+- [Link](https://currency-exchange-clnb.onrender.com/)
+
+- [Video](https://www.loom.com/share/130cd4b846b2427b913277ff44560791)
```

Comment:
- [ ] Your video presentation failed to show the piece of code you are proud of making or find interesting. Kindly record your video again with a demo of the app, as you have done already along with your VS code or editor showing your piece of code you like. Kindly review the video before submitting and ensure it makes the requirement. The video should also not exceed 5 minutes in length.